Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -124,14 +143,26 @@ extension BrowserViewController {
             UIKeyCommand(input: "\\", modifierFlags: [.command, .shift], action: #selector(showTabTrayKeyCommand)), // Safari on macOS
             UIKeyCommand(input: "\t", modifierFlags: [.command, .alternate], action: #selector(showTabTrayKeyCommand), discoverabilityTitle: Strings.showTabTrayFromTabKeyCodeTitle)
         ]
+        
+        let bookmarkEditing = [
+            UIKeyCommand(input: "g", modifierFlags: [.command], action: #selector(addBookmarkCommand))
```

Comment:
it's still incorrect

CMD + D -> add bookmark
CMD + G -> find next
CMD + Shift + G -> find previous